tests/tensor_ops/test_setitem.py

```diff
@@ -11,6 +14,7 @@
     _is_bool_array_index,
     _is_int_array_index,
 )
+from tests.custom_strategies import arbitrary_indices
 
 from ..custom_strategies import adv_integer_index, basic_indices, broadcastable_shapes
```

Comment:
any reason you separate the imports from `custom_strategies`?

Reply:
Nope! Fixed